octopuslite/transform.py

```diff
@@ -17,21 +17,25 @@ def __call__(self, x, idx):
         if self.transforms is None:
             return x
 
-        tform = tf.AffineTransform(translation=self.transforms[idx, ...])
+        #tform = tf.AffineTransform(translation=self.transforms[idx, ...])
+        tform = tf.AffineTransform(translation=self.transforms[idx, :2, 2])
         return tf.warp(x, tform, preserve_range=True)
 
 
 
-def parse_transforms(path: os.PathLike, n: int) -> StackTransformer:
+def parse_transforms(path: os.PathLike,
+                    #n: int
+                    ) -> StackTransformer:
     """Parse a file or folder containing registration transforms.
 
     Parameters
     ----------
     path : PathLike
         A path to either a numpy file of transforms, or a folder containing
         transforms as XML files.
-    n : int
-        The number of frames in the movie.
+    # # may become necessary when alternate transform format is used
+    # n : int
+    #     The number of frames in the movie.
```

Comment:
Rather than comment out `n` can you just remove it?

Reply:
done ✅